femmecubator/open-sprints
# Scaffold main page

#### What does this PR do?
- Scaffolds landing page

#### How should this be manually tested?
`yarn && yarn start` & verify home page resembles [mock](https://www.figma.com/file/QlZDY3OZn7u2X0R67V7g31/Project%3A-Open-Sprint?node-id=67%3A2697&mode=dev) 

#### Any background context you want to provide?
- A lot of Docusaurus defaults still need to be pruned, but I think it's ok to keep them for now so that we have examples to refer to as we build.
- Previously we decided that we were going to use `styled-components` for styling, but I chose to go with CSS Modules, as I discovered that Docusaurus's support for CSS-in-JS is still on shaky ground.
